docs/ref/settings.txt

```diff
@@ -428,6 +428,15 @@ any hyphens with underscores, and adding an ``'HTTP_'`` prefix to the name.
 For example, if your client sends a ``'X-XSRF-TOKEN'`` header, the setting
 should be ``'HTTP_X_XSRF_TOKEN'``.
 
+CSRF_TRUSTED_ORIGINS
+----------------
+
+.. versionadded:: 1.9
+
+Default: ``[]``
+
+An iterable containing domains which will be included as part of the referer checks during CSRF processing of secure requests.
```

Comment:
"A list of hosts which are trusted origins for POST requests. By default, Django's CSRF protection requires that under HTTPS a POST request must come from your same domain. If you need cross-origin POST requests, include the source hosts in this setting. Don't include any host in this setting that you don't control or trust to not CSRF your users!"